src/pineko/evolve.py

```diff
@@ -101,11 +101,7 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     operators_card = copy.deepcopy(default_card)
     x_grid, _pids, _mur2_grid, muf2_grid = pineappl_grid.axes()
     sv_method = sv_scheme(tcard)
-    xif = (
-        1.0
-        if (sv_method == "expanded" or sv_method == "exponentiated")
-        else tcard["XIF"]
-    )
+    xif = 1.0 if sv_method in ["expanded", "exponentiated"] else tcard["XIF"]
```

Comment:
```suggestion
    xif = 1.0 if sv_method is not None else tcard["XIF"]
```

You can assume the runcard has been already validated, so the type is faithful.

However, in general, with enum you can always load them from strings, e.g. with `ScVarMethod["expanded"]`, if it fails is not a member of the enum :)
(but in this case assume is one of the allowed values, and thus check against `None`)